src/CoreWCF.Primitives/src/CoreWCF/Security/XmlHelper.cs

```diff
@@ -411,5 +411,40 @@ public static UniqueId ReadTextElementAsUniqueId(XmlElement element)
         {
             return new UniqueId(ReadTextElementAsTrimmedString(element));
         }
+
+        internal static string[] TokenizeInclusiveNamespacesPrefixList(string inclusiveNamespacesPrefixList)
+        {
+            if (inclusiveNamespacesPrefixList == null)
+                return null;
+
+            string[] prefixes = inclusiveNamespacesPrefixList.Split(null);
+            int count = 0;
```

Comment:
You can simplify this code by using the Split overload which takes StringSplitOptions and pass RemoveEmptyEntries. This way the returned array will always be the correct length and you only need to replace "#default" with String.Empty. That allows you to drop the prefix.Length check and the resize code at the end.